box/boxunreal.py

```diff
@@ -5,6 +5,7 @@
 from ue5osc import Communicator
 
 from .boxnavigator import Action, BoxNavigatorBase
+from .box import Pt
```

Comment:
Where is `Pt` used?

Reply:
def sync_box_position_to_unreal(self) -> None:
        """Move Boxsim agent to match Unreal Agent Position"""
        unrealX, unrealY, _ = self.ue5.get_location()
        target = Pt(unrealX, unrealY)
        self.navigator.position = target